src/applications/mhv-landing-page/utilities/data/index.js

```diff
@@ -118,6 +118,15 @@ const resolveLandingPageLinks = (
     },
   ];
 
+  const paymentsLinks = [
+    HEALTH_TOOL_LINKS.PAYMENTS[0],
```

Comment:
I will suggest a different approach to this instead of using indexing ([0], [1]). Perhaps a splice just to add the new entry in the second location.